legacy/Makefile.include

```diff
@@ -74,6 +75,7 @@ CFLAGS   += $(OPTFLAGS) \
             -DRAND_PLATFORM_INDEPENDENT=1 \
             -I$(TOOLCHAIN_DIR)/include \
             -I$(TOP_DIR) \
+            -I$(TOP_DIR)/.. \
```

Comment:
This looks like it's adding the root directory of the repository to the include paths. If that is the case, then why? We shouldn't need to be doing that.

Reply:
Adding symlink to the root vendor directory fixes it. See e981bc7f1bed31e9f349aaf1b77c803a1f474f20.